samples/Nancy.Demo.Hosting.AspnetCore/Startup.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Nancy.Owin;
+
+namespace Nancy.Demo.Hosting.AspnetCore
+{
+    public class Startup
+    {
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
+        {
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        {
+	        if (env.IsDevelopment())
+		        app.UseDeveloperExceptionPage();
+
+	        app.UseOwin().UseNancy();
+
+			app.Run(async (context) =>
+            {

```

Comment:
The indentation is a bit off here

Reply:
Why even include this block of code? The Nancy middleware should handle the request, so this should never be hit.